pkg/cmd/cli/cmd/rollback.go

```diff
@@ -210,24 +213,31 @@ func (o *RollbackOptions) Run() error {
 			return err
 		}
 		target = deployment
+		configName = r.Name
```

Comment:
How can I add a default prior to this since I don't know what the input object is?